app/assets/javascripts/distributions_and_transfers.js

```diff
@@ -94,11 +94,39 @@ $(function() {
     }
   );
 
-  $(function() {
+  $(function () {
     if (storage_location_required && !control.val()) {
       $("#__add_line_item").addClass("disabled");
     }
 
     request_storage_location_and_populate_item(default_item);
   });
+
+  $('#new_distribution').submit(function (e) {
+    e.preventDefault();
+
+    const year = $('select#distribution_issued_at_1i option:selected').val();
+    const month = $('select#distribution_issued_at_2i option:selected').val();
+    const day = $('select#distribution_issued_at_3i option:selected').val();
+
+    if (year && month && day) {
+      let submit;
+      const asked = $(this).data('asked');
+
+      const selectedDate = new Date(year, month - 1, day);
+      const currentDate = new Date();
```

Comment:
```suggestion
      const defaultDateInput = new Date();
```
Could we change this from `currentDate` to something like `defaultDateInput` since I think it loses a little context here. Or maybe a function to encapsulate this conditional?

Reply:
Sure, it looks better. Thanks!